<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-D8HNM4S" ignored="0" name="Test2" tests="3" failures="1" timestamp="2020-11-22T00:59:17 IST" time="33.398" errors="0">
  <testcase name="launchSite" time="27.878" classname="tests.AmazonLoginTest"/>
  <testcase name="loginOperation" time="5.353" classname="tests.AmazonLoginTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.AmazonLoginTest.loginOperation(AmazonLoginTest.java:152)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- loginOperation -->
  <testcase name="closeSite" time="0.115" classname="tests.AmazonLoginTest"/>
</testsuite> <!-- Test2 -->
